app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -78,20 +78,20 @@ def diagnostic_codes
   end
 
   def prepare_for_evss!
-    return if pending_eps? || disability_not_service_connected?
+    return if pending_eps? || disabilities_not_service_connected?
 
     save_metadata(forward_to_mas_all_claims: true) if Flipper.enabled?(:rrd_mas_all_claims_tracking) &&
                                                       !single_issue_hypertension_cfi?
 
     if Flipper.enabled?(:rrd_mas_disability_tracking) && single_disability_eligible_for_mas?
       save_metadata(forward_to_mas: true)
-      insert_classification_codes
+      insert_classification_codes unless Flipper.enabled?(:rrd_mas_all_claims_notification)
     end
   end
 
   def send_post_evss_notifications!
     send_completed_notification if rrd_job_selector.rrd_applicable?
-    notify_mas
+    conditionally_notify_mas
```

Comment:
Rename suggested in a [previous PR](https://github.com/department-of-veterans-affairs/vets-api/pull/10916#discussion_r997272462)